src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java

```diff
@@ -122,11 +122,11 @@ private String getResultIcon(Result result) {
         if (result == Result.SUCCESS) {
             return ":white_check_mark:";
         } else if (result == Result.ABORTED) {
-            return ":point_up:";
+            return "";
```

Comment:
Maybe a warning icon ⚠️  would also be appropriate: http://emojipedia.org/warning-sign/